src/gei/AzureApiFactory.cs

```diff
@@ -0,0 +1,36 @@
+using System.Net.Http;
+
+namespace OctoshiftCLI.GithubEnterpriseImporter
+{
+    public sealed class AzureApiFactory : IAzureApiFactory
+    {
+        private readonly OctoLogger _octoLogger;
+        private readonly IHttpClientFactory _clientFactory;
+        private readonly IBlobServiceClientFactory _blobServiceClientFactory;
+        private readonly EnvironmentVariableProvider _environmentVariableProvider;
+
+        public AzureApiFactory(OctoLogger octoLogger, IHttpClientFactory clientFactory, EnvironmentVariableProvider environmentVariableProvider, IBlobServiceClientFactory blobServiceClientFactory)
+        {
+            _octoLogger = octoLogger;
+            _clientFactory = clientFactory;
+            _environmentVariableProvider = environmentVariableProvider;
+            _blobServiceClientFactory = blobServiceClientFactory;
+        }
+
+        public AzureApi Create(string azureStorageConnectionString)
```

Comment:
```suggestion
        public AzureApi Create(string azureStorageConnectionString = null)
```

Reply:
Updated in https://github.com/github/gh-gei/pull/221/commits/266f0d07425b14207ad953a1fb8661746c973ed1